composer/algorithms/utils/augmentation_primitives.py

```diff
@@ -105,7 +113,7 @@ def autocontrast(pil_img: Image.Image, level: float = 0.0):
     .. seealso:: :func:`PIL.ImageOps.autocontrast`.
 
     Args:
-        pil_img (Image.Image): The image
+        pil_img (:class:`Image.Image`): The image.
```

Comment:
```suggestion
        pil_img (:class:`PIL.Image.Image`): The image.
```

I think `PIL.` is needed here (and all subsequent references) for the type link to be rendered properly ([see docs here](https://mosaicml-composer--999.com.readthedocs.build/en/999/api_reference/composer.algorithms.utils.augmentation_primitives.html?highlight=augmentation_primitives))

Reply:
Got it, I didn't realize cross-linking was enabled for PIL. I'll add those from here on out.